lms/static/js/spec/student_account/register_spec.js

```diff
@@ -143,10 +170,10 @@
                                 defaultValue: '',
                                 type: 'select',
                                 options: [
-                                {value: '', name: '--'},
-                                {value: 1900, name: '1900'},
-                                {value: 1950, name: '1950'},
-                                {value: 2014, name: '2014'}
+                                    {value: '', name: '--'},
+                                    {value: 1900, name: '1900'},
+                                    {value: 1950, name: '1950'},
+                                    {value: 2014, name: '2014'}
```

Comment:
Hm, I don't remember even doing this. Think my IDE might have done it automatically. Aw well.